.pylintrc

```diff
@@ -0,0 +1,16 @@
+[MASTER]
+extension-pkg-allow-list=
+    clr
+
+ignore-long-lines=yes
+
+disable=
```

Comment:
That's a lot of disabled rules!  Any of these things we should really be fixing instead of ignoring?  Just checking.

Reply:
Yes, we can fix for all of warnings, but we don't need to. Pylint is just assistance, not a law.
Maybe this article is more explanatory: 
 - [What Pylint is not?](https://pylint.pycqa.org/en/latest/intro.html#what-pylint-is-not)

And FYI and me to remember in future:
 - warning codes can be searched with `pylint --list-msgs | grep <warning-name>`